lib/utils/get_catergoric.dart

```diff
@@ -41,6 +41,9 @@ List<String> getCategoric(WidgetRef ref) {
 
   Map<String, Type> roles = ref.read(typesProvider);
 
+
+  // Watching stdout to get variables that are Ignored.
```

Comment:
Why do you need to watch rather than read?

Reply:
Thanks. I know `Use ref.watch to make your widgets react to changes in the state, and ref.read when you need to perform actions or access state without triggering rebuilds.` In this case, change this part to `ref.read`.